domains/transfer-agreements/TransferAgreements.API/API/Controllers/TransferAgreementsController.cs

```diff
@@ -37,4 +39,21 @@ public async Task<ActionResult> Create([FromBody] CreateTransferAgreement reques
 
         return Created($"api/transfer-agreements/{result.Id}", result);
     }
+
+    [ProducesResponseType(200)]
+    [HttpGet("api/transfer-agreements")]
+    public async Task<ActionResult<List<TransferAgreementResponse>>> GetBySubjectId()
+    {
+        var subject = User.FindSubjectClaim();
+
+        if(!Guid.TryParse(subject, out var subjectId))
+        {
+            return BadRequest("Invalid subject ID.");
+        }
+
+        var response = await transferAgreementRepository.GetTransferAgreementsBySubjectId(subjectId);
```

Comment:
Is there a reason why this endpoint only returns the transfer agreements where the user is "Sender" and not where the user is "Receiver"?